src/krux/pages/addresses.py

```diff
@@ -91,7 +91,7 @@ def list_address_type(self, addr_type=0):
                     self.ctx.display.clear()
                     self.ctx.display.draw_centered_text(loading_txt % (num_checked + 1))
 
-                    pos_str = str(num_checked + 1) + ". "
+                    pos_str = str(num_checked + 1) + "." + " "  # thin space
```

Comment:
I'm not seeing the difference in these two values.  Is it just to be more explicit that the space is needed, so it is not carelessly removed in the future?

Reply:
Thin space is used as a trick. Krux line breaker will split words detecting spaces, but "thin spaces" won't be detected. On this case Tadeu found out multisig address wouldn't fit on Dock's screen, thin space will ensure the address number prefix will be in the same line as the address start (not in a exclusive line) so the screen space will be better used and address will fit.

Reply:
@jdlcdl here you can see the unicode character: https://www.compart.com/en/unicode/U+2009

Reply:
Nice! I didn't even realize it was not a normal space character.  Thank you for the explanation.